odml/tools/rdf_converter.py

```diff
@@ -0,0 +1,141 @@
+import sys
+import uuid
+
+from rdflib import Graph, BNode, Literal, URIRef
+from rdflib.namespace import XSD, RDF
+
+import odml
+import odml.format as format
```

Comment:
how about `odml_format` because I have already used the `fmt` inside `save_element()`?